code/apps/Managed Software Center/Managed Software Center/Controllers/MainWindowController.swift

```diff
@@ -292,6 +294,15 @@ class MainWindowController: NSWindowController, NSWindowDelegate, WKNavigationDe
     
     func loadInitialView() {
         // Called by app delegate from applicationDidFinishLaunching:
+        
+        // add custom link if nessesary
+        if let CustonSidebarItem = pref("CustonSidebarItem") as? Dictionary<String, String> {
```

Comment:
Need to change the name to "CustomSidebarItem" in all occurrences.

Reply:
I must have missed it :)

Reply:
It's "CustonSidebarItem" pretty much everywhere, and should be "CustomSidebarItem".

Reply:
OMG... you're right... I've been working on it for too long...